pages/create.js

```diff
@@ -0,0 +1,120 @@
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+import { uid } from "uid";
+import styled from "styled-components";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledLabel = styled.label`
+  font-family: var(--fontBold);
+  font-size: 18px;
+`;
+
+const StlyedInput = styled.input`
+  width: 100%;
+  background-color: white;
+  padding: 10px;
+  margin-block: 7px 20px;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  font-family: var(--fontRegular);
+`;
+
+const StyledTextArea = styled.textarea`
+  width: 100%;
+  height: 100px;
+  background-color: white;
+  padding: 10px;
+  margin-block: 7px 20px;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  font-family: var(--fontRegular);
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledSubmitButton = styled.button`
+  border: none;
+  border-radius: 5px;
+  width: 49%;
+  padding: 10px 40px;
+  margin-top: 20px;
+  background-color: var(--accentColor);
+  color: white;
+  font-size: 16px;
+  box-shadow: 0px 1px 3px var(--primaryFontColor);
+  &:hover {
+    background-color: #5d8a55;
+    cursor: pointer;
+  }
+`;
+
+const StyledCancelLink = styled(Link)`
+  border: none;
+  display: inline-block;
+  border-radius: 5px;
+  width: 49%;
+  padding: 10px 40px;
+  margin-top: 20px;
+  background-color: darkgrey;
+  color: var(--primaryBackgroundColor);
+  text-align: center;
+  text-decoration: none;
+  box-shadow: 0px 1px 3px var(--primaryFontColor);
+  &:hover {
+    background-color: grey;
+    cursor: pointer;
+  }
+`;
```

Comment:
There seem to be a fair number of repeating styles here. 

It might be worth looking into [extending styes with styled components](https://styled-components.com/docs/basics#extending-styles).

Reply:
Thanks for sharing the Link! We created a Button Component now with different Buttons/Links so we don't have to repeat our styles anymore.